client/libra-dev/Cargo.lock

```diff
@@ -0,0 +1,1922 @@
+# This file is automatically @generated by Cargo.
```

Comment:
Is there a particular reason why libra-dev is in its own workspace? It probably makes more sense to keep everything in one workspace?

Reply:
We need libra-dev in its own workspace because FFI needs 

# FFI libray can only support panic = "abort"
[profile.dev]
panic = 'abort'

[profile.release]
panic = 'abort'
lto = true

to be part of the cargo.toml. Since profiles are ignored for any sub-crates, we need libra-dev to be in the same layer as the main workspace. Unfortunately profile overrides do not work for panic, lto, or rpath settings.